Sprint-1/exercises/decimal.js

```diff
@@ -7,3 +7,9 @@ const num = 56.5678;
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
 // Log your variables to the console to check your answers
+
+const wholeNumberPart=parseInt(num);
+const decimalPart=(num%1).toFixed(4);// const decimalPart=(num-Math.floor(num)).toFixed(4);
```

Comment:
Think about how you might be able to modify the expression to get the decimal part so that it would work for any value of num, even if it has a different number of digits. Could you make use of your previous result?

Reply:
I have used the previous expressions for the decimalPart.